cli/src/main/java/software/amazon/smithy/java/cli/SmithyCall.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.java.cli;
+
+import java.io.ByteArrayOutputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.concurrent.Callable;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+import picocli.CommandLine.Parameters;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.ArgGroup;
+import software.amazon.smithy.java.aws.client.auth.scheme.sigv4.SigV4AuthScheme;
+import software.amazon.smithy.java.aws.client.awsjson.AwsJson1Protocol;
+import software.amazon.smithy.java.aws.client.core.identity.EnvironmentVariableIdentityResolver;
+import software.amazon.smithy.java.aws.client.core.settings.RegionSetting;
+import software.amazon.smithy.java.aws.client.restjson.RestJsonClientProtocol;
+import software.amazon.smithy.java.aws.client.restxml.RestXmlClientProtocol;
+import software.amazon.smithy.java.client.core.auth.scheme.AuthSchemeResolver;
+import software.amazon.smithy.java.client.core.endpoint.EndpointResolver;
+import software.amazon.smithy.java.client.core.interceptors.ClientInterceptor;
+import software.amazon.smithy.java.client.core.interceptors.RequestHook;
+import software.amazon.smithy.java.client.rpcv2.RpcV2CborProtocol;
+import software.amazon.smithy.java.core.serde.ShapeSerializer;
+import software.amazon.smithy.java.core.serde.document.Document;
+import software.amazon.smithy.java.dynamicclient.DynamicClient;
+import software.amazon.smithy.java.json.JsonCodec;
+import software.amazon.smithy.model.Model;
+import software.amazon.smithy.model.shapes.OperationShape;
+import software.amazon.smithy.model.shapes.ShapeId;
+
+@Command(name = "smithy-call", mixinStandardHelpOptions = true, version = "1.0",
+        description = "Smithy Call, send request to Smithy services using Smithy models")
+public final class SmithyCall implements Callable<Integer> {
+    private static final JsonCodec CODEC = JsonCodec.builder().build();
+    private static final Logger LOGGER = Logger.getLogger(SmithyCall.class.getName());
+
+    private static final String[] BASE_RESOURCE_FILES = {
+            "aws.api.smithy",
+            "aws.auth.smithy",
+            "aws.customizations.smithy",
+            "aws.protocols.smithy"
+    };
+
+    @Parameters(index = "0", description = "Service Name (e.g. com.example#CoffeeShop)")
+    private String service;
+
+    @Parameters(index = "1", description = "Name of the operation to perform on the service", arity = "0..1")
```

Comment:
This is required right? Why is the arity then `0..1` instead of 1. I think you can just skip specifying the arity.